april heartbeat (#2372)
iterative/dvc.org: content/blog/2021-04-15-april-21-dvc-heartbeat.md

﻿---
title: April '21 Heartbeat
date: 2021-04-15
description: |

>   Monthly updates are here! Loads from the Community, use cases, videos, 5 new hires and more!

Reviewer: 💅 please try to keep under 80 symbols

descriptionLong: |
  Monthly updates are here! Read all about what our community is doing
  with DVC, our CEO's interviews and talks, new team members and more!
picture: 2021-04-15/apr21cover.png
author: jeny_defigueiredo

> commentsUrl: https://discuss.dvc.org/t/april-21-heartbeat/718?u=jendefig

Reviewer: `?u=jendefig` - no need for this

tags:
  - Heartbeat
  - DVC
  - MLOps
---

## From the Community

We're starting with the community this month because it has been overflowing
with great content from our users. It's like we're on a sugar high!

![Sugar High](https://media.giphy.com/media/oiGCnybFPh6Q8/giphy.gif)

> ### Goku Mohandas Tutorial

Reviewer: I think he calls them lessons (it's more of a course) - we can use the same terminology. May be mentioned that it is part of the course

First up, [Goku Mahandas](https://twitter.com/GokuMohandas) of
[Made With ML](https://madewithml.com/) has created

> [this tutorial](https://madewithml.com/courses/mlops/versioning/) using DVC.  
> It's RT'ing around the MLOps Twitter space like hotcakes! 🥞

Reviewer 2: Was that new line intentional?
```suggestion
[this tutorial](https://madewithml.com/courses/mlops/versioning/) using DVC.
It's RT'ing around the MLOps Twitter space like hotcakes! 🥞
```

Reviewer 2: p.s. breaks the p:
![image](https://user-images.githubusercontent.com/1477535/114788876-dbc08c00-9d47-11eb-8a2c-ab6667d5e97b.png)

<external-link
href="https://madewithml.com/courses/mlops/"
title="MLOps"

> description="DVC tutorial"

Reviewer: could we put some better descriptions in these items please?

  [Harshit Tyagi](https://twitter.com/dswharshit) of
  [Data Science with Harshit](https://www.youtube.com/channel/UCH-xwLTKQaABNs2QmGxK2bQ),
  and
- Dmitry's [talk](https://www.youtube.com/watch?v=J8mCr3wVgdA) at the
  [Toronto Machine Learning Society](https://twitter.com/TMLS_TO)

Spoiler alert ⚠️: You can't choose wrong!

![](https://media.giphy.com/media/GXrcAztzRX9kI/giphy.gif)

## And we keep on growing our worldwide team! 🌏

We are getting to the point where our new hires could take up our whole
Heartbeat! 😅🚀💗

[**Julie Galvan**](https://www.linkedin.com/in/julianna-galvan/) joins our team
from Houston, Texas as an engineer. She is focused on web development. In her
free time Julie loves reading, especially fantasy fiction (Harry Potter #6 was
fav) and paper crafting. Welcome Julie!🎉

[**Matt Seddon**](https://www.linkedin.com/in/matt-seddon/) joins us from Down
Under as a DVC front-end engineer! 🦘 He lives in Kiama, a small town on the
East Coast of Australia. Originally from Scotland, when he's not programming he
likes to spend time with his family away from screens (😅🙌🏼) and he volunteers
for the state emergency service. 🤲🏼

> [**Yanxiang Gao**](https://www.linkedin.com/in/gaoyanxiang/)(who graciously

Reviewer: 💅 need space before `(who ...`

﻿---
title: April '21 Heartbeat
date: 2021-04-15
description: |
  Monthly updates are here! Loads from the Community, use cases, videos, 

>   5 new hires and more!

Reviewer 2: 💅 
```suggestion
  5 new hires, and more!
```

Author: You guys are really comma happy. Ok.  I will submit.

Author: Also added comment after member in long version for consistency.  😉

Reviewer 2: Haha. I usually avoid going into stylistic suggestions but I thought "5 new hired and more" sounded a bit like "more than 5 new hires".

descriptionLong: |
  Monthly updates are here! Read all about what our community is doing

>   with DVC, our CEO's interviews and talks, new team members and more!

Reviewer 2: Didn't expand on use cases or videos here.

Reviewer 2: BTW I'm not sure where short vs long descriptions display. Is the short one for SEO? If so it should have a max length, probably? (which I think we're within here, just curious what are the guidelines here)

Author: I think the short description is on the main blog page post card and the long description is at the top when you click into it.  Not sure which works with SEO 🤷🏻‍♀️ @rogermparent, can you help us with SEO and how we might be thinking about this in this respect?

Author: Let me know if you think this still needs further extending @jorgeorpinel !

Reviewer 3: > the short description is on the main blog page post card and the long description is at the top when you click into it.

Yep! Also, all the SEO meta tags use the short description. As far as the content, the only big thing to consider when making a description from this standpoint is to prioritize making a good description that draws attention over stuffing it with keywords (though some keywords in the right place can draw attention!), as Google doesn't consider meta description keywords in its ranking algorithm.

![seo-screenshot](https://user-images.githubusercontent.com/9111807/115066881-206e3380-9ebe-11eb-9044-4a795d9f8a80.png)

Reviewer 2: > the short description is on the main blog page post card and the long description is at the top when you click into it

Ah yes now that the PR is deploying I can confirm that. Thanks for the reminder! 

> Not sure which works with SEO 🤷🏻‍♀️ 

Yep. It looks like the short one will go into social media previous (good to know). As for Google search results, they do whatever they want anyway — sometimes they use the suggested title/description, sometimes they build their own.

BTW (since we're tagging @rogermparent 😬) maybe for the featured (latest) post we can use the long one even in /blog ? There's space to spare, it seems:

![image](https://user-images.githubusercontent.com/1477535/115067099-0d576580-9eb6-11eb-85ff-ca2bee556a43.png)

Reviewer 2: p.s. just noticed your answer after posting mine, @rogermparent.

Reviewer 2: > making a good description that draws attention over stuffing it with keywords

As long as they match the content yes, otherwise the algo may even penalize ranking for trying to mislead it 😋 or so I've heard.

Author: > BTW (since we're tagging @rogermparent 😬) maybe for the featured (latest) post we can use the long one even in /blog ? There's space to spare, it seems:

I agree, there's definitely space for it, I can change.  @rogermparent, on the short description for SEO purposes are there length guidelines?  What = short?

Reviewer 2: > What = short?

Probably whatever fits comfortably in our blog cards will roughly work for social media previews too:

![image](https://user-images.githubusercontent.com/1477535/115068620-1a755400-9eb8-11eb-9d52-aa59b55dc7fd.png)

Reviewer 3: > BTW (since we're tagging @rogermparent grimacing) maybe for the featured (latest) post we can use the long one even in /blog ? There's space to spare, it seems:
> 
> ![image](https://user-images.githubusercontent.com/1477535/115067099-0d576580-9eb6-11eb-85ff-ca2bee556a43.png)

That's a good idea, we should be able to fetch the long description of the first article in the query and show it on the first card of these index pages. If nothing else, we can make a ticket for the future.

Reviewer 2: https://github.com/iterative/dvc.org/issues/2386

picture: 2021-04-15/april21cover.png
author: jeny_defigueiredo
commentsUrl: https://discuss.dvc.org/t/april-21-heartbeat/718
tags:
  - Heartbeat
  - DVC
  - MLOps
  - Made with ML
  - AI Singapore
  - DAGsHub
---

## From the Community

We're starting with the community this month because it has been overflowing
with great content from our users. It's like we're on a sugar high!

![Sugar High](https://media.giphy.com/media/oiGCnybFPh6Q8/giphy.gif)

### Goku Mohandas' New Lessons!

First up, [Goku Mahandas](https://twitter.com/GokuMohandas) of

> [Made With ML](https://madewithml.com/) has created
> [this tutorial](https://madewithml.com/courses/mlops/versioning/) using DVC.  
> It's RT'ing around the MLOps Twitter space like hotcakes! 🥞

Reviewer 2: Per https://github.com/iterative/dvc.org/pull/2372#discussion_r613693094:

```suggestion
[Made With ML](https://madewithml.com/) has added this
[Versioning](https://madewithml.com/courses/mlops/versioning/) lesson to the
MLOps course using DVC. It's RT'ing around the ML Twitter space like hotcakes!
🥞
```

Reviewer 2: p.s. please retweet some hotcakes my way.

Author: Much better.  Thank you!  I'm correcting this way on my side and not committing suggestion because that's how I usually get in to trouble.  Trying to get this out the door..... Maybe that makes this more complicated?

﻿---
title: April '21 Heartbeat
date: 2021-04-16
description: |
  Monthly updates are here! Loads from the Community, use cases, videos, 
  5 new hires, and more!
descriptionLong: |

>   Monthly updates are here! Read all about new lessons and a tutorials from

Reviewer: btw, just a tip - this section supports links, and _different_ **text** `styles`

[Made With ML](https://madewithml.com/) has added this
[Versioning Lesson](https://madewithml.com/courses/mlops/versioning/) to the
popular **MLOps Course** using DVC.  
It's RT'ing around the MLOps Twitter space like hotcakes! 🥞

<external-link
href="https://madewithml.com/courses/mlops/"
title="MLOps - Versioning Code, Data and Models"
description="Using DVC to version data and models for reproducibility 
in a local storage use case"
link="https://madewithml.com/"
image="/uploads/images/2021-04-15/made-with-ml-logo.png"/>

### Ryzal Kamis Tutorial

[**Ryzal Kamis**](https://www.linkedin.com/in/ryzalkamis/) of
[AI Singapore](https://twitter.com/AISingapore) has created an
[**in depth tutorial**](https://makerspace.aisingapore.org/2021/04/data-versioning-for-cd4ml-part-2/)
on data versioning using DVC. This is a follow up article to his
[tutorial that was featured in the September Heartbeat.](https://dvc.org/blog/september-20-dvc-heartbeat)
Thanks Ryzal for this detailed work! 🙏🏼

<external-link
href="https://makerspace.aisingapore.org/2021/04/data-versioning-for-cd4ml-part-2/"
title="Data Versioning for CD4ML - Part 2"

> description="DVC tutorial"

Reviewer: we still have the same `DVC tutorial` description in all these snippets (except the first one). Could we please make them a bit longer and specific to the item?

﻿---

> title: April '21 Heartbeat

Reviewer: Merged, but I see that we should also change the file name ^^

and images like `/uploads/images/2021-04-15/made-with-ml-logo.png` - I'm not even sure why it does work now
